yarn.lock

```diff
@@ -5209,8 +5209,8 @@ locate-path@^2.0.0:
     path-exists "^3.0.0"
 
 lodash-es@^4.17.4:
-  version "4.17.5"
-  resolved "https://registry.yarnpkg.com/lodash-es/-/lodash-es-4.17.5.tgz#9fc6e737b1c4d151d8f9cae2247305d552ce748f"
+  version "4.17.7"
+  resolved "https://registry.yarnpkg.com/lodash-es/-/lodash-es-4.17.7.tgz#db240a3252c3dd8360201ac9feef91ac977ea856"
```

Comment:
I don't mind to include that, but I want to be sure it isn't an accident :)

Reply:
Yeah, I'm super confused by `yarn`. This was the result of invoking `yarn upgrade mattermost-redux`. If I run an unconstrained `yarn upgrade`, of course, I get a whole lot more updates. It's not obvious to me if yarn thinks this is required...

I based the decision to keep it on @enahum's update to my PR on https://github.com/mattermost/mattermost-mobile/pull/1469/commits/ebc8a43bfff53676c73fe479640eba4978288a2c which changed the `mime-types` dependency as well.

I'm 0/5 on keeping it, but it was intentional -- of sorts :)

Reply:
I tried reading up on how `yarn upgrade` and `yarn install` works recently, and it's kind of confusing how they sometimes just upgrade stuff without you asking it to. I think stuff like this just happens sometimes